src/ocm/components/clusters/ClusterStatus.tsx

```diff
@@ -48,15 +50,16 @@ export const ClusterStatusIcon: React.FC<ClusterStatusIconProps> = ({ status, ..
   }
 };
 
-export const getClusterStatusText = (status: Cluster['status']) =>
-  CLUSTER_STATUS_LABELS[status] || status;
+export const getClusterStatusText = (status: Cluster['status'], t: TFunction) =>
```

Comment:
This function is exported and used in `selectors/clusters` where it's missing the `t` prop.